protected/views/element/update.php

```diff
@@ -144,60 +134,75 @@
 JS
 );
 
-
 $attributes = [];
 foreach ($model->colorAttributes() as $attribute) {
     $attributes[$attribute] = [
         'type' => Form::INPUT_HTML5,
         'html5type' => 'color',
-        'allowUnsafe' => true,
     ];
 }
 
+$this->registerCss(<<<CSS
+.columns .form-group {
+    break-inside: avoid;
+    margin-bottom: 0.25rem;
+}
+.columns label {
+    display: inline-block;
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+CSS
+);
+
+echo Html::beginTag('div', ['style' => ['column-width' => '250px'], 'class' => ['columns']]);
 if (!empty($attributes)) {
-    $form->formConfig['labelSpan'] = 8;
     echo Form::widget([
         'form' => $form,
         'model' => $model,
-        'columns' => 2,
         'attributes' => $attributes
     ]);
 }
-echo ButtonGroup::widget([
-    'options' => [
-        'class' => [
-            'pull-right',
-            'buttons-row'
-        ],
+echo Html::endTag('div');
+
+echo Form::widget([
+    'form' => $form,
+    'model' => $model,
+    'attributes' => [
+        FormButtonsWidget::embed([
+            'buttons' => [
+                [
+                    'label' => $this->title,
+                    'type' => ButtonGroup::TYPE_SUBMIT,
+                    'style' => 'primary',
+                    'buttonOptions' => [
+                        'name' => 'action',
+                        'value' => 'refresh'
+                    ],
+                ],
+                [
+                    'label' => $model->isNewRecord
+                        ? \Yii::t('app', 'Create element & go back', ['action' => $this->title])
```

Comment:
This might need cleanup, is this view actually used for new elements?

Reply:
Yes it is used for both create and update.